operator/internal/manifests/internal/config/options.go

```diff
@@ -46,11 +44,9 @@ type WriteAheadLog struct {
 	IngesterMemoryRequest int64
 }
 
-// Value calculates the floor of the division of
-// querier cpu limits to the query frontend replicas
-// available.
-func (p Parallelism) Value() int32 {
-	return int32(math.Floor(float64(p.QuerierCPULimits) / float64(p.QueryFrontendReplicas)))
+// Value returns the number of available CPU cores per querier.
+func (p MaxConcurrent) Value() int32 {
```

Comment:
Now that there is no calculation needed anymore, does it make sense to directly use the attribute value and get rid of this function?

If we want to keep the function:
- Do we want to bound this value to a certain minimum/maximum?
- I'm unsure of the coding style: Does it need to be moved up one block to be grouped with its type or are all methods grouped together?